tdm-be/package.json

```diff
@@ -18,27 +18,28 @@
   "author": "",
   "license": "CeCILL",
   "dependencies": {
-    "axios": "^1.6.2",
+    "axios": "^1.6.7",
     "cors": "^2.8.5",
+    "express": "^4.18.2",
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Gosh!

That was in `devDependencies`!?
Better now :+1:

Reply:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏We should use precise versions

So that sequential builds always produce same versions installed.
Oh.
Wait.
`npm ci` is used?
So, it uses `package-lock.json`.
OK, no need to make packages versions precise.